### PR TITLE
Move boilerplate to shared location

### PR DESCRIPTION
**Pull request by enj**
/hold

4.2

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Eventually going to make it to `library-go`? Or probably going to stay external?

**Comment by enj** [comment]
> Eventually going to make it to `library-go`? Or probably going to stay external?

Not anytime soon.

File: pkg/operator2/operator.go [hunk 1]
```diff
@@ -10,6 +10,9 @@ import (
 	"strconv"
 	"strings"
 
+	"monis.app/go/openshift/controller"
+	"monis.app/go/openshift/operator"
```
Comment (wking) [inline pkg/operator2/operator.go]: Hmm, @enj do these need to get moved back into an OpenShift repo (library-go?) in the wake of #205?

Comment (enj) [reply pkg/operator2/operator.go]: https://github.com/openshift/operator-boilerplate-legacy

**Pull request merged**